swf/src/write.rs

```diff
@@ -1821,8 +1821,9 @@ impl<W: Write> Writer<W> {
     }
 
     fn write_bevel_filter(&mut self, filter: &BevelFilter) -> Result<()> {
-        self.write_rgba(&filter.shadow_color)?;
+        // Note that the color order is wrong in the spec, it's hightlight then shadow.
```

Comment:
Should we add this to our errata in the wiki?

Reply:
Also there's a typo here.

Reply:
Also I assume reading is already done in this order, and the actual appearance of the filter is expected. Right?

Reply:
Added to wiki and fixed typo! Reading is good yeah.